.github/workflows/clang-tidy.yml

```diff
@@ -0,0 +1,51 @@
+name: Clang-tidy check
+on:
+  schedule:
+    - cron:  '0 6 * * 1-5'
```

Comment:
Does clang-tidy need to be on cron too?

Reply:
Yeah, the idea is that it will check the code every night and if it finds something it will create an issue and a PR with  fixes if possible.

Reply:
Got it, thanks!